ee/tabby-db/src/lib.rs

```diff
@@ -156,6 +157,15 @@ impl DbConn {
 
     /// Initialize database, create tables and insert first token if not exist
     async fn init_db(pool: SqlitePool) -> Result<Self> {
+        let user_input: String = input(
+            "The database is from an older version. Type 'confirm' to run database migrations:",
+        )
+        .interact()?;
```

Comment:
we shall only prompt user input when the sqlx migration version in db doesn't match the embed migration's version - thus here we need to check the version before prompting users.